README.md

```diff
@@ -1,8 +1,59 @@
-# vscode-aks-tools
+# Azure Kubernetes Service (AKS) Extension for Visual Studio Code (Preview)
 
-* View your AKS clusters in the Kubernetes extension cloud explorer
-* Add AKS clusters to kubeconfig via the GUI
-* Run AppLens detector diagnostics on your AKS clusters
+The Visual Studio Code extension for Azure Kubernetes Service helps you view your AKS cluster, merge and save kubeconfig, perform AKS Diagnostics checks, and run AKS Periscope within VS Code.
+
+## Prerequisites
+
+Ensure that you have installed the [Kubernetes Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) extension for VS Code prior to installing the Azure Kubernetes Service Extension for Visual Studio Code following these [instructions](https://code.visualstudio.com/docs/azure/kubernetes#_install-the-kubernetes-extension).
+
+## Installation
+
+1. Download and install the [Azure Kubernetes Service extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) for Visual Studio Code.
+
+2. Wait for the extension to finish installing then reload Visual Studio Code when prompted.
+
+3. Once the installation is complete, you'll see a section named **Azure** under **Clouds**.
+
+4. Sign in to your Azure Account by clicking **Sign in to Azure…**
```

Comment:
:bulb: This is really cool, but if we are detailing this, should we add screenshots, like mentioned here: https://github.com/Azure/vscode-aks-tools/pull/6 Sharing just for consensus, and putting the idea out there.

Reply:
+1 yeah, a screenshot to highlight the Kubernetes activity bar icon and point up the Clouds section would be useful.  Also to help people find the sign-in command.

Reply:
Added a screenshot